src/pointer.cr

```diff
@@ -491,6 +491,30 @@ struct Pointer(T)
     Slice.new(self, size)
   end
 
+  # Returns a `Slice` from a given null-terminated pointer array, with an optional,
+  # though recommended, limit.
+  #
+  # ```
+  # ptr = LibC.getgrnam("wheel").value.gr_mem
+  # slice = pointer.to_null_terminated_slice(LibC::NGROUP_MAX)
+  # ```
+  def to_slice_null_terminated(limit : Int)
+    count = 0
+    while (!self[count].null?)
+      raise("Limit reached without NULL terminator.") if (count > limit)
+      count += 1
+    end
+    return to_slice(count)
+  end
+
+  def to_slice_null_terminated
```

Comment:
Missing `ditto` comment to copy the documentation.

Reply:
Fixed